### PR TITLE
Fix/fix compatibility with xcode8

### PR DESCRIPTION
**Pull request by TroubleMakerBen**

### CODE REVIEW

File: MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m [hunk 1]
```diff
@@ -457,36 +449,57 @@ - (nullable NSURL *)buildTokenRequestURLWithAppSecret:(NSString *)appSecret rele
...
+      if (error.code == 1) {
+        MSLogError([MSDistribute logTag], @"Authentication session was cancelled by user or failed.");
+      }
+      if (callbackUrl) {
+        [self openURL:callbackUrl];
+      }
+    };
+
+    // Initialize the SFAuthenticationsession.
+    typedef void (*MSInitSFAuthenticationSession)(id, SEL, NSURL *, NSString *, MSCompletionBlockForAuthSession);
```
Comment (clpolet) [inline MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m]: I love it!

File: Config/Global.xcconfig [hunk 2]
```diff
@@ -17,6 +17,7 @@ PRODUCT_NAME = $(PROJECT_NAME)
 PUBLIC_HEADERS_FOLDER_PATH = include/$(PRODUCT_NAME)
 
 // :Mark: Language configurations
+
```
Comment (clpolet) [inline Config/Global.xcconfig]: Is it normal not to have the `CLANG_MODULES_AUTOLINK = NO` here?

Comment (jaelim-ms) [reply Config/Global.xcconfig]: We don't need that because we stay to build the SDK on Xcode 8.

Comment (jaelim-ms) [reply Config/Global.xcconfig]: @TroubleMakerBen Probably you can remove the empty line. 😄

Comment (clpolet) [reply Config/Global.xcconfig]: OK so we Bitrise is back to XCode 8 I guess.

**Pull request merged**